app/helpers/emoji_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module EmojiHelper
+  def emoji_codes_to_img(str)
+    if str
+      str = str.gsub(/:([\w\-+]*(?::t\d)?):/) do
+        code = $1
+        "<img src=\"#{Emoji.url_for(code)}\" title=\"#{code}\" class=\"emoji\" alt=\"#{code}\">"
```

Comment:
To avoid injection, perhaps we shouldn't render anything if the `url_for(code)` doesn't find an emoji?

That way there is no harm injecting the `code` into the string like that.

Reply:
In this case this is supposed to be escaped, but I do agree we should prevent it from the source

Reply:
I am with Robin that we should support

```
EmojiHelper.emoji_codes_to_img("a test :bogus: a test") => "a test :bogus: a test"
```

If we don't find a URL just gsub with yourself. 

Besides that this is good to merge imo.